tests/unit/test_darray.c

```diff
@@ -1,7 +1,5 @@
-/**
- * @file
+/*
```

Comment:
You've tried to simplify this test and that's fine, but instead of replacing a working test why not create a new one?